### PR TITLE
[#14] Fix Database Migration

### LINKED ISSUE #14: Fix Database Migration
Getting the following error when trying to run the database migration.
```
rake aborted!
NameError: uninitialized constant GalterIrExporterMigrationSurveyItems
```

### PR DESCRIPTION
**Pull request by Meowcenary**
Change generated installation migration name to resolve name bug. Add
development steps and author to README. closes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,17 @@ Pass the `--help` option to view the ways to override the exported fields for yo
 
 TODO: Add when [#7](https://github.com/galterlibrary/galter_ir_exporter/issues/7) is resolved.
 
+### DigitalHub Development
+
+Branch from master, make any changes to the gem, then push the gem to this repository. Once that is done clone and
```
Comment (sharpattack) [inline README.md:49]: Can we make the following change? 
`push the gem to this repository` -> `push your branch to this repository`

File: lib/generators/galter_ir_exporter/install/install_generator.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ class Install < Rails::Generators::Base
 
     # Setup the database migrations
     def copy_migrations
-      migration_template "create_galter_ir_exporter_migration_survey_items.rb", "db/migrate/galter_ir_exporter_migration_survey_items.rb"
+      migration_template "create_galter_ir_exporter_migration_survey_items.rb", "db/migrate/create_galter_ir_exporter_migration_survey_items.rb"
```
Comment (sharpattack) [inline lib/generators/galter_ir_exporter/install/install_generator.rb:14]: I am glad I passed this task onto you. You saved me from spending a lot of time investigating a different idea I had in mind.

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ And then execute:
 
 Run the installer, creating the necessary migration files:
 
-    $ bundle exec rails galter_ir_exporter:install
+    $ bundle exec rails generate galter_ir_exporter:install
```
Comment (sharpattack) [inline README.md:19]: 🤦 Thanks.

File: README.md [hunk 4]
```diff
@@ -44,6 +44,17 @@ Pass the `--help` option to view the ways to override the exported fields for yo
 
 TODO: Add when [#7](https://github.com/galterlibrary/galter_ir_exporter/issues/7) is resolved.
 
+### DigitalHub Development
+
+Branch from master, make any changes to the gem, then push the gem to this repository. Once that is done clone and
+bundle [DigitalHub/digital-repository](https://github.com/galterlibrary/digital-repository). Once everything is
+installed add the following to the digital-repository Gemfile:
```
Comment (sharpattack) [inline README.md:51]: I may remove the reference to digitalhub/digital-repository. That is specific to us. Instead, I may remove the "clone and bundle" step, and say something simple like the following:
`Then, add the following to your samvera/sufia6 app's Gemfile`

File: galter_ir_exporter.gemspec [hunk 5]
```diff
@@ -5,7 +5,7 @@ require "galter_ir_exporter/version"
 Gem::Specification.new do |spec|
   spec.name          = "galter_ir_exporter"
   spec.version       = GalterIrExporter::VERSION
-  spec.authors       = ["Austin Sharp"]
+  spec.authors       = ["Austin Sharp", "Eric Newman"]
```
Comment (sharpattack) [inline galter_ir_exporter.gemspec:8]: 🎉 🎉 🎉

File: README.md [hunk 6]
```diff
@@ -44,6 +44,17 @@ Pass the `--help` option to view the ways to override the exported fields for yo
 
 TODO: Add when [#7](https://github.com/galterlibrary/galter_ir_exporter/issues/7) is resolved.
 
+### DigitalHub Development
```
Comment (sharpattack) [inline README.md:47]:
```suggestion
### Testing Your Changes In Your Samvera/Sufia6 App
```

File: README.md (hunk 4 above)
Comment (Meowcenary) [reply README.md:51]: Sure. I meant this as temporary getting started with development instructions, but I'm not really sure what the proper steps are for developing a new gem.

File: README.md (hunk 1 above)
Comment (Meowcenary) [reply README.md:49]: Sure thing.

**Comment by Meowcenary** [comment]
Suggestions have been committed. I'm going to merge this now.

**Pull request merged**